force-app/main/default/lwc/nksConversationNoteViewer/nksConversationNoteViewer.js

```diff
@@ -0,0 +1,118 @@
+    themeGroup;
+    isRead;
+    apiName;
+    activeFlowApiName;
+    showFlow = false;
+    conversationNoteSubscription = null;
+
+    connectedCallback() {
+        this.subscribeToMessageChannel();
+    }
+
+    disconnectedCallback() {
+        this.unsubscribeToMessageChannel();
+    }
+
+    @wire(MessageContext)
+    messageContext;
+
+    @wire(getRecord, { recordId: '$recordId', fields: CONVERSATION_NOTE_FIELDS })
+    wiredRecord({ error, data }) {
+        if (data) {
+            this.conversationNote = getFieldValue(data, CONVERSATION_NOTE_FIELD);
+            this.themeGroup = getFieldValue(data, THEME_GROUP_FIELD);
+            this.isRead = getFieldValue(data, IS_READ_FIELD);
+        } else if (error) {
+            console.error('Error fetching Conversation Note:', error);
+        }
+    }
+
+    handleJournal() {
+        this.openFlow('NKS_Conversation_Note_Journal_Case_v_2');
+    }
+
+    handleRedact() {
+        this.openFlow('Conversation_Note_Set_Redaction');
+    }
+
+    handleCreateTask() {
+        this.openFlow('STO_Conversation_Note_Send_NAV_Task');
+    }
+
+    openFlow(flowApiName) {
+        this.showFlow = false;
+        // eslint-disable-next-line @lwc/lwc/no-async-operation, @locker/locker/distorted-window-set-timeout
+        setTimeout(() => {
+            this.activeFlowApiName = flowApiName;
+            this.showFlow = true;
+        }, 10);
+    }
+
+    handleFlowStatusChange(event) {
+        const { status, outputVariables } = event.detail;
+        if (status === 'FINISHED') {
+            handleShowNotifications('journal_conversation', outputVariables, this.notificationBoxTemplate, true);
+            this.showFlow = false;
+        }
+    }
+
+    subscribeToMessageChannel() {
+        if (!this.conversationNoteSubscription) {
+            this.conversationNoteSubscription = subscribe(
+                this.messageContext,
+                CONVERSATION_NOTE_NOTIFICATIONS_CHANNEL,
+                (message) => this.handleMessage(message),
+                { scope: APPLICATION_SCOPE }
+            );
+        }
+
+        if (!this.buttonContainerSubscription) {
+            this.buttonContainerSubscription = subscribe(
+                this.messageContext,
+                BUTTON_CONTAINER_NOTIFICATIONS_CHANNEL,
+                (message) => this.handleMessage(message),
+                { scope: APPLICATION_SCOPE }
+            );
+        }
+    }
+
+    unsubscribeToMessageChannel() {
+        if (this.conversationNoteSubscription) {
+            unsubscribe(this.conversationNoteSubscription);
+            this.conversationNoteSubscription = null;
+        }
+
+        if (this.buttonContainerSubscription) {
+            unsubscribe(this.buttonContainerSubscription);
+            this.buttonContainerSubscription = null;
+        }
+    }
+
+    get getTitle() {
+        return 'Samtalereferat ' + (this.themeGroup ? this.themeGroup : '') + ' - ' + (this.isRead ? this.isRead : '');
+    }
+
```

Comment:
Æ trur vi har et formel felt for det der, NKS_Timeline_Name__c, det e ikke helt likt men mulig vi skal endre det for nytt design anyways?

Reply:
Takk! Visste ikke om det. Jeg bruker det feltet nå, så har jeg sendt en melding til Lena og spurt om vi skal bruke det eller lage et nytt :) Jeg synes egentlig det så ganske bra ut med å bare bruke NKS_Timeline_Name__c som det er i dag.